src/modules/auth/components/login/loginForm/loginForm.tsx

```diff
@@ -28,7 +32,12 @@ export function LoginForm() {
   }
 
   function onSubmit(data: LoginFormData) {
-    console.log(data);
+    axiosInstance
+      .post('/auth/login', data)
+      .then(() => navigate('/'))
```

Comment:
Create another constant file in the same manner as in routes I described earlier, and use that file for the further addition of the routes. Reason for the that is the same, routes might change.

Reply:
fixed